Server/VikingVault.DataAccess/Migrations/VikingVaultDbContextModelSnapshot.cs

```diff
@@ -19,6 +19,31 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                 .HasAnnotation("Relational:MaxIdentifierLength", 128)
                 .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
 
+            modelBuilder.Entity("VikingVault.DataAccess.Models.AutomaticPayment", b =>
+                {
+                    b.Property<int>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasAnnotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn);
+
+                    b.Property<float>("Amount");
+
+                    b.Property<int>("CompanyId");
+
+                    b.Property<DateTime>("InitialPaymentDate");
+
+                    b.Property<DateTime>("LastPaymentDate");
+
+                    b.Property<int?>("PayingUserId");
```

Comment:
Should property `PayingUserId` be nullable?

Reply:
If _PayingUserId_ would not be nullable, then it should have the _[Required]_ annotation above. But if both foreign keys: Company and PayingUser (which are of the same type (User)) have the  _[Required]_ annotation, the database will not be updated, and the following message appears : System.Data.SqlClient.SqlException: 'Introducing FOREIGN KEY constraint 'FK_AutomaticPayments_User_PayingUserId' on table 'AutomaticPayments' may cause cycles or multiple cascade paths. Specify ON DELETE NO ACTION or ON UPDATE NO ACTION, or modify other FOREIGN KEY constraints.
Could not create constraint or index. See previous errors.'

Therefore, I decided to remove the annotation on PayingUser, hence, this column is nullable.